app/services/weet.js

```diff
@@ -23,3 +26,24 @@ exports.createWeet = async weet => {
     throw databaseError(error);
   }
 };
+
+exports.getAllWeets = async params => {
+  try {
+    const { rows: weets, count } = await Weet
+      .findAndCountAll({
+        ...params,
+        attributes: { exclude: [USERID_PARAM] },
```

Comment:
Why exclude this?

Reply:
because the user object already has an id. If I don't exclude it:

![image](https://user-images.githubusercontent.com/31294096/143035085-7840798b-e9fb-4648-983d-a308c06f8491.png)